src/client/dfs/dfs.c

```diff
@@ -562,14 +562,14 @@ insert_entry(daos_handle_t oh, daos_handle_t th, const char *name, size_t len,
 	sgl.sg_iovs	= sg_iovs;
 
 	rc = daos_obj_update(oh, th, flags, &dkey, 1, &iod, &sgl, NULL);
-	if (rc) {
-		/** don't log error if conditional failed */
-		if (rc != -DER_EXIST && rc != -DER_NO_PERM)
+
+	if ( (rc != 0) && (rc != -DER_EXIST) ) {
```

Comment:
(style)  space prohibited after that open parenthesis '('
(style)  space prohibited before that close parenthesis ')'